src/main/resources/db/migration/V020__RemoveRecordingUrl.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE public.recordings
```

Comment:
I think this migration may need to come in in a second PR. Reason being the pipeline to production runs migrations _before_ code is deployed. Meaning that any select queries generated by spring boot will fail if the column name is used explicitly (I'm not sure how it works at a low level, maybe it does *) it'll fail once this is dropped and the new code is yet to be deployed (have seen this take up to 5mins for flex to apply the change)

So yeh, I'd actually drop this migration out of the PR, release this PR then ad the migration as a tidy up when you're 100% certain the column is no longer used

Reply:
Removed migration + created pr: https://github.com/hmcts/pre-api/pull/532 to be merged after this pr is merged